package.json

```diff
@@ -34,35 +34,35 @@
     "@asset-pipe/js-reader": "^2.3.4",
     "@asset-pipe/sink-fs": "1.1.4",
     "@asset-pipe/sink-mem": "1.1.3",
-    "@metrics/client": "2.1.1",
-    "abslog": "2.2.3",
+    "@metrics/client": "2.4.1",
+    "abslog": "2.4.0",
     "body": "5.1.0",
     "bole": "3.0.2",
     "boom": "7.3.0",
-    "compression": "1.7.3",
-    "convict": "4.4.1",
+    "compression": "1.7.4",
+    "convict": "5.1.0",
     "cors": "2.8.5",
-    "express": "4.16.4",
+    "express": "4.17.1",
     "fast-json-stable-stringify": "2.0.0",
-    "jest-worker": "23.2.0",
-    "joi": "14.3.0",
-    "mime-types": "2.1.21",
-    "p-retry": "3.0.0",
-    "parse-json": "4.0.0",
-    "readable-stream": "3.1.0",
-    "uuid": "3.3.2"
+    "jest-worker": "24.9.0",
+    "joi": "14.3.1",
+    "mime-types": "2.1.24",
+    "p-retry": "4.1.0",
+    "parse-json": "5.0.0",
+    "readable-stream": "3.4.0",
+    "uuid": "3.3.3"
   },
   "devDependencies": {
-    "eslint": "^5.10.0",
-    "eslint-config-finn": "^3.0.0",
-    "eslint-config-finn-prettier": "^3.0.1",
-    "eslint-plugin-import": "^2.8.0",
-    "husky": "^1.3.0",
-    "jest": "^23.2.0",
-    "lint-staged": "^8.1.0",
-    "prettier": "^1.15.3",
+    "eslint": "^6.4.0",
+    "eslint-config-finn": "^3.0.1",
+    "eslint-config-finn-prettier": "^3.0.2",
+    "eslint-plugin-import": "^2.18.2",
+    "husky": "^3.0.5",
```

Comment:
You need to update the husky config. `precommit` as an npm script doesn't work with Husky 3

Reply:
Or just remove huskey...

Reply:
I went for this approach: remove lint-staged and then reconfigure husky to just check to make sure you don't have lint errors when you push so you avoid the annoyance of builds failing when you forget to lint. I like this because then you get to keep your commits nice and quick and it's only 1x when you push that you get some slow down and that short slowdown is likely to save you a good bit of time since you wont end up with builds breaking because of lint issues.

Reply:
Works for me